telemetry/definitions/commonDefinitions.json

```diff
@@ -297,6 +350,11 @@
             "type": "string",
             "description": "The ID of the job currently running"
         },
+        {
+            "name": "codeTransformLocalBuildSystemVersion",
```

Comment:
I believe something like this was added in https://github.com/aws/aws-toolkit-common/pull/783

Please try to think of types that are more generic. There is no reason to have : 


```
codeTransformLocalBuildSystemVersion
someOtherFeatureLocalBuildSystemVersion
antherFeatureLocalBuildSystemVersion
```

Please, always apply this principle when designing metrics. Redundant, overlapping definitions makes it hard to find useful metrics, both when using them as well as visualizing them in the backend.

Reply:
I discussed with @dhasani23 on #783 and we will include the same changes in this PR (and close that one). I'll change this to `localBuildSystemVersion`